### PR TITLE
[E2E-Test]: add Test_EIBC_Client_NoFundForToken_EVM

### PR DESCRIPTION
**Pull request by hungdinh82**

### CODE REVIEW

File: tests/eibc_client_test.go [hunk 1]
```diff
@@ -1044,3 +1044,296 @@ func Test_EIBC_Client_Lower_Fee_EVM(t *testing.T) {
...
+	require.NoError(t, err)
+
+	rollappHeight, err := rollapp1.GetNode().Height(ctx)
+	require.NoError(t, err)
+
+	// Assert balance was updated on the hub
+	// testutil.AssertBalance(t, ctx, rollapp1, rollappUserAddr, rollapp1.Config().Denom, walletAmount.Sub(transferData.Amount))
+
+	// wait until the packet is finalized
+	isFinalized, err := dymension.WaitUntilRollappHeightIsFinalized(ctx, rollapp1.GetChainID(), rollappHeight, 300)
```
Comment (anhductn2001) [inline tests/eibc_client_test.go:1209]: We dont need wait anymore @hungdinh82

File: tests/eibc_client_test.go [hunk 2]
```diff
@@ -1044,3 +1044,296 @@ func Test_EIBC_Client_Lower_Fee_EVM(t *testing.T) {
...
+	require.NoError(t, err)
+	require.True(t, isFinalized)
+
+	txhash, err := dymension.GetNode().FinalizePacketsUntilHeight(ctx, dymensionUserAddr, rollapp1.GetChainID(), fmt.Sprint(rollappHeight))
```
Comment (anhductn2001) [inline tests/eibc_client_test.go:1213]: not need to finalize packets too

File: tests/eibc_client_test.go [hunk 3]
```diff
@@ -1044,3 +1044,296 @@ func Test_EIBC_Client_Lower_Fee_EVM(t *testing.T) {
...
+	require.True(t, isFinalized)
+
+	_, err = dymension.GetNode().FinalizePacketsUntilHeight(ctx, dymensionUserAddr, rollapp1.GetChainID(), fmt.Sprint(rollappHeight))
+	require.NoError(t, err)
+
+	err = testutil.WaitForBlocks(ctx, 5, dymension, rollapp1)
+	require.NoError(t, err)
+
+	// Minus 0.1% of transfer amount for bridge fee
+	testutil.AssertBalance(t, ctx, dymension, dymensionUserAddr2, rollappIBCDenom, transferData.Amount.Sub(bridgingFee).Sub(eibcFee))
```
Comment (anhductn2001) [inline tests/eibc_client_test.go:1335]: not sub eibcFee because order didn't fulfilled

**Comment by hungdinh82** [comment]
Close #430

File: tests/eibc_client_test.go [hunk 4]
```diff
@@ -1044,3 +1044,271 @@ func Test_EIBC_Client_Lower_Fee_EVM(t *testing.T) {
...
+	channel, err := ibc.GetTransferChannel(ctx, r, eRep, dymension.Config().ChainID, rollapp1.Config().ChainID)
+	require.NoError(t, err)
+
+	err = r.StartRelayer(ctx, eRep, ibcPath)
+	require.NoError(t, err)
+
+	err = testutil.WaitForBlocks(ctx, 10, dymension, rollapp1)
+	require.NoError(t, err)
+
+	err = testutil.WaitForBlocks(ctx, 10, dymension, rollapp1)
```
Comment (anhductn2001) [inline tests/eibc_client_test.go:1190]: remove it, not need

File: tests/eibc_client_test.go [hunk 5]
```diff
@@ -1044,3 +1044,271 @@ func Test_EIBC_Client_Lower_Fee_EVM(t *testing.T) {
...
+	require.NoError(t, err)
+
+	rollappHeight, err := rollapp1.GetNode().Height(ctx)
```
Comment (anhductn2001) [inline tests/eibc_client_test.go:1193]: same above

File: tests/eibc_client_test.go [hunk 6]
```diff
@@ -1044,3 +1044,271 @@ func Test_EIBC_Client_Lower_Fee_EVM(t *testing.T) {
...
+	require.NoError(t, err)
+
+	err = testutil.WaitForBlocks(ctx, 5, dymension, rollapp1)
```
Comment (anhductn2001) [inline tests/eibc_client_test.go:1196]: remove too

**Pull request closed without merging**